### PR TITLE
data: Add several new Lenovo PIDs

### PR DESCRIPTION
**Pull request by jigpu**
Received information through private bug reports about several new Lenovo
PIDs that were requested for addition.

Signed-off-by: Jason Gerecke <jason.gerecke@wacom.com>

### CODE REVIEW

File: data/isdv4-51b0.tablet [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Lenovo ThinkPad X13 Yoga
+# Sensor Type: AES
+# Features: Touch (Integrated), Tilt
+# HW Resolution: 29376 x 16524 (2540 x 2540 lpi)
+#
+# Autogenerated from sysinfo.wym3nbQqZN
```
Comment (whot) [inline data/isdv4-51b0.tablet]: I think we should reference where we can find this, otherwise it just looks like a leftover.

Comment (jigpu) [reply data/isdv4-51b0.tablet]: Would a link to https://github.com/linuxwacom/wacom-hid-descriptors/ suffice? I really don't want to have to dynamically generate URLs.

Comment (jigpu) [reply data/isdv4-51b0.tablet]: Added a link; re-open this thread if that's not enough

Comment (whot) [reply data/isdv4-51b0.tablet]: yes, sorry, email got drowned out. That link is sufficient, thanks.

**Pull request merged**